package.json

```diff
@@ -300,6 +300,7 @@
     "sinon": "^7.0.0"
   },
   "dependencies": {
+    "@folio/react-intl-safe-html": "^1.0.200010",
```

Comment:
This version does not exist outside CI. If there's a bug fix in react-intl-safe-intl we need to depend on, let's get it released and depend on a proper, released version here.

Reply:
Actually, NBD, but dealt with in #514 / [UIIN-500](https://issues.folio.org/browse/UIIN-500)